js/index.js

```diff
@@ -269,11 +273,12 @@ initBoardEvListener();
 
 // Add 'click' event-listener to the 'New Game' button.
 const newGameButton = document.getElementById("new-game");
-newGameButton.addEventListener("click", () =>
-    console.log(
-        "New Game button: Click Event Triggered"
-    )
-);
+newGameButton.addEventListener("click", () => {
+    console.log("New Game button: Click Event Triggered");
```

Comment:
Line `277` can be removed since we are no longer testing the function.

Reply:
`console.log` removed.